mm2src/coins/eth/web3_transport/websocket_transport.rs

```diff
@@ -46,13 +46,13 @@ pub struct WebsocketTransport {
     node: WebsocketTransportNode,
     event_handlers: Vec<RpcTransportEventHandlerShared>,
     pub(crate) proxy_sign_keypair: Option<Keypair>,
-    controller_channel: Arc<ControllerChannel>,
+    controller_channel: ControllerChannel,
     connection_guard: Arc<AsyncMutex<()>>,
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
Why replacing `Arc` with `Clone`?

Reply:
both `tx` & `rx` inside were `Arc`ed which made me think we extract/clone them out of `ContollerChannel` so wrapping with a bigger arc is just a complex optimization.
this doesn't seem the case though (tx already clonable, rx was never cloned out in the first place).
[8ae149b](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2291/commits/8ae149bc194c4bab48a146a8b373e9a74e52306e)